packages/types/src/payment/common.ts

```diff
@@ -417,6 +417,11 @@ export interface PaymentSessionDTO {
    */
   data: Record<string, unknown>
 
+  /**
+   * Payment session context
+   */
+  context?: Record<string, unknown>
```

Comment:
discussion: in the DTO's that reflect a public model API I don't think we should have `?` but we can take care of that at a later point